### PR TITLE
Smol Fixes

### PR DESCRIPTION
**Pull request by Arokha**
Running PR for commits that fix little things. See commit names for changes.

### CODE REVIEW

File: code/modules/power/gravitygenerator_vr.dm [hunk 1]
```diff
@@ -93,6 +93,7 @@ GLOBAL_LIST_EMPTY(gravity_generators)
 	. = ..()
 	setup_parts()
 	middle.add_overlay("activated")
+	overlay_state = "activated"
```
Comment (TheFurryFeline) [inline code/modules/power/gravitygenerator_vr.dm:96]: Undefined var.

File: maps/tether/tether-05-station1.dmm [hunk 2]
```diff
@@ -8051,9 +8051,8 @@
 	opacity = 0
 	},
 /obj/machinery/door/airlock/engineering{
-	name = "Engineering Hard Storage";
-	req_access = list(11);
-	req_one_access = newlist()
+	name = "Gravity Generator";
+	req_access = list(11)
```
Comment (TheFurryFeline) [inline maps/tether/tether-05-station1.dmm:8055]: Also may want to add an air alarm for the little hallway between the Grav Genny and maints.

Comment (Arokha) [reply maps/tether/tether-05-station1.dmm:8055]: But there's no pumps or anything for it to control

Comment (TheFurryFeline) [reply maps/tether/tether-05-station1.dmm:8055]: Then add 'em? :3

Comment (Arokha) [reply maps/tether/tether-05-station1.dmm:8055]: it's a tiny 2x2 room with a single light bulb, it doesn't even have an area, it's not worth it

**Pull request merged**